examples/js/loaders/GLTF2Loader.js

```diff
@@ -2234,8 +2234,21 @@ THREE.GLTF2Loader = ( function () {
 
 									for ( var j = 0, jl = positionAttribute.array.length; j < jl; j ++ ) {
 
-										positionAttribute.array[ j ] += position.array[ j ];
-
+									    // vertex saved in single buffer view
+                                        // p0,n0,t0,w0,p1,n1,t1,w1,...pn,nn,tn,wn in buffer
+                                        // so we need calculate offset & stride
+                                        if (position.data !== undefined) {
+                                            var index = Math.floor(j / position.itemSize) * position.data.stride + position.offset;
+                                            positionAttribute.array[ j ] += position.array[ j%position.itemSize +index];
+                                        }
+                                        // vertex saved in separate buffer
+                                        // view
+                                        // p0,p1,p2...pn,
+                                        // n0,n1,n2...nn,
+                                        // t0,t1,t2...tn
+                                        else {
+                                            positionAttribute.array[ j ] += position.array[ j ];
+                                        }
```

Comment:
Just guessing without a model to check, but I think this can be simplified to handle any BufferAttribute or InterleavedBufferAttribute type:

```js
for ( var j = 0, len = positionAttribute.count(); j < len; j++ ) {

  positionAttribute.setXYZ( 
    j,
    positionAttribute.getX( j ) + position.getX( j ),
    positionAttribute.getY( j ) + position.getY( j ),
    positionAttribute.getZ( j ) + position.getZ( j )
  );

}
```

(and similarly below for normals)